rosbag2_storage/package.xml

```diff
@@ -14,6 +14,8 @@
 
   <depend>pluginlib</depend>
   <depend>rcpputils</depend>
+  <depend>rcl_interfaces</depend>
```

Comment:
I am not in favor of this change - from a top level design perspective `rosbag2_storage` is intended to be as close as possible to ROS-agnostic. All parameter parsing should happen in `rosbag2_transport` where all explicit interaction with ROS 2 happens. I imagine there we could just have specific helper functions that deal with params and spit out `StorageOptions`/`RecordOptions`/etc..